### PR TITLE
Fix command chain

### PR DESCRIPTION
**Pull request by vihanb**
### I have:
 - [x] Ran a test (`npm run dev && npm test`)
 - [x] Added tests for my changes
 - [x] Documented my code thoroughly.

### CODE REVIEW

**Comment by somebody1234** [comment]
Um okay the `1a` test fails because IMO there are other cases where this being allowed is much better

**Comment by vihanb** [comment]
@somebody1234 is it good idea to allow commad chain head to be ID or literal or should it just be id

File: test/Parser/expressions.js [hunk 1]
```diff
@@ -46,4 +52,11 @@ export default () => describe("Expressions", () => {
         
         invalid `a[c]?d`
     });
+    
+    describe("Closure Command Chains", () => {
+        valid`if foo {bar}`;
```
Comment (vihanb) [inline test/Parser/expressions.js:58]: Ono y if as command chain?

**Comment by somebody1234** [comment]
IMO literal should be allowed as well, so things like `'mystring' reverse` will be valid

File: src/vsl/parser/vsltokenizer.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ export default class VSLTokenizer extends Tokenizer {
             ['\\$+[0-9]+', passThrough, VSLTokenType.SpecialArgument],
             ['\\$[a-zA-Z_][a-zA-Z0-9_]*', slice1, VSLTokenType.SpecialIdentifier],
             ['\\.[0-9_]+', strip_, VSLTokenType.Decimal],
-            ['[0-9][0-9_]*\\.[0-9_]+', strip_, VSLTokenType.Decimal],
+            ['[0-9][0-9_]*\\.[0-9_]*', strip_, VSLTokenType.Decimal],
```
Comment (vihanb) [inline src/vsl/parser/vsltokenizer.js:84]: Does this allow `1. + 1.` to be valid? This will break `1.foo` no?

**Review by vihanb — COMMENTED** [review]
Just do a `git fetch origin && git merge develop` and fix conflict and good to go

**Comment by vihanb** [comment]
@somebody1234 can you check the Travis logs it's erroring due to no `expr` variable defined

**Comment by somebody1234** [comment]
IDK how to fix that, working fine on c9, maybe something to do with different versions of node?

**Comment by vihanb** [comment]
@somebody1234 it's failing on all versions. Did you update the package-lock.json

**Comment by somebody1234** [comment]
Um help it's working fine on c9

**Pull request closed without merging**